### ISSUE TITLE
[v0.5.0-alpha] Clipping issue

### ISSUE THREAD

**Issue by JuiceWarrior** [Teeyenoh/zephis#28]
The grass texture used for the player model has a bad habit of being wider than most tiles in the game, meaning passing next to something causes the grass tile to phase through it at the sides rather unconvincingly.

The grass doesn't even move when it clips for crying out loud :P

**Comment by JuiceWarrior** [comment]
This is a big problem with the gap in the farthest rock wall from the starting point, where the clipping is very obvious

**Comment by JuiceWarrior** [comment]
Ok, scratch that, the entire rock wall is passable. That's another issue

**Comment by Teeyenoh** [comment]
Right, I've noticed what you mean, I guess all I can say now is that most sprites won't be as wide as the grass sprite (it's designed to be a tile after all) and clipping shouldn't occur very much :-) keep an eye on it for me though :-)

**Comment by JuiceWarrior** [comment]
This is still an issue in v0.5.3.1-alpha. No visible changes have been made with this update
